_posts/hi/newsletters/2022-11-02-newsletter.md

```diff
@@ -0,0 +1,211 @@
+---
+title: 'Bitcoin Optech Newsletter #224'
+permalink: /hi/newsletters/2022/11/02/
+name: 2022-11-02-newsletter-hi
+slug: 2022-11-02-newsletter-hi
+type: newsletter
+layout: newsletter
+lang: hi
+---
+इस सप्ताह का समाचार पत्र वैकल्पिक रूप से नोड्स को पूर्ण RBF को सक्षम करने की अनुमति देने के बारे में
+निरंतर चर्चा का वर्णन करता है, BIP324 संस्करण 2 एन्क्रिप्टेड ट्रांसपोर्ट प्रोटोकॉल के एक डिजाइन तत्व पर
+प्रतिक्रिया के लिए अनुरोध करता है, LN विफलताओं और विशेष नोड्स के लिए देरी को विश्वसनीय रूप से
+जिम्मेदार ठहराने के लिए एक प्रस्ताव का सारांश देता है, और लिंक करता है आधुनिक LN HTLCs के लिए एंकर
+आउटपुट का उपयोग करने के विकल्प के बारे में चर्चा। नए सॉफ़्टवेयर रिलीज़ और रिलीज़
+उम्मीदवारों की घोषणाओं के साथ हमारे नियमित अनुभाग भी शामिल हैं --- LNडी के लिए एक सुरक्षा महत्वपूर्ण
+अद्यतन --- और लोकप्रिय Bitcoin Infrastructure सॉफ़्टवेयर में उल्लेखनीय परिवर्तनों के विवरण शामिल हैं।
+
+## समाचार
+
+- **mempool स्थिरता:** Anthony Towns ने Bitcoin-Dev मेलिंग सूची पर एक [चर्चा] शुरू की
+  [कस्बों की स्थिरता] लेनदेन रिले और mempool स्वीकृति के लिए Bitcoin Core की नीतियों को
+  कॉन्फ़िगर करना आसान बनाने के परिणामों के बारे में, जैसे कि `mempoolfullrbf` विकल्प के योग द्वारा
+  किया गया था Bitcoin Core की विकास शाखा (see न्यूज़लेटर्स [#205][news205 rbf],
+  [#208][news208 rbf], [#222][news222 rbf], और [#223][news223 rbf])।
```

Comment:
Can you make the small change here so we can link to the 223/hi version?

https://github.com/bitcoinops/bitcoinops.github.io/pull/923#discussion_r1018199424